packages/falco-bin/PKGBUILD

```diff
@@ -4,35 +4,38 @@
 # Contributor: Kris Nóva R.I.P. <kris@nivenly.com>
 
 pkgbase=falco-bin
-pkgname=("falco-bin"
-         "falco-bin-dkms")
+pkgname=falco-bin
+provides=(falco)
+conflicts=(falco)
+backup=('etc/falco/falco_rules.yaml'  'etc/falco/falco.yaml')
```

Comment:
the rules are not provided with the binaries, they have to be installed with falcoctl now

Reply:
But the `falcoctl` it's a separated package, imho or we add as dependencies the `falcoctl` here or we ship falco with some default rules and let the users use `falcoctl` if they want to download them from the oci artifacts registry and put them in any path they like.
Wdyt? 
In arch each package should be kind of self contained and also the `falcoctl` needs to be executed with sudo to write into the `/etc/falco/falco_rules.yaml`, `backup` will let save the new files as `.pacnew` in case they have been modified externally the falco package.

Reply:
better for the UX to embed the latest tagged version of the rules

Reply:
done